courses/models.py

```diff
@@ -1,3 +1,45 @@
 from django.db import models
 
+from users.models import AppUser
+
 # Create your models here.
+
+#Front-End Base Model
+
+class Course(models.Model):
+    course_code = models.CharField(max_length=7)
+    course_title = models.TextField(blank=False)
+    fall_offering = models.BooleanField()
+    spring_offering = models.BooleanField()
+    summer_offering = models.BooleanField()
+    PENG_required = models.BooleanField()
+
+    class Meta:
+        managed = True  #auto creates tables
+        db_table = 'Courses'
+
+
+
+
+#Algorithm Specific Models
+
+
+
+# class CourseSection(models.Model):
```

Comment:
Is this commented-out code needed for something later?

Reply:
Yeah its for the Algorithm stuff that Im working on this sprint